banners/thank_you/translations_de.js

```diff
@@ -1,12 +1,19 @@
 export const translations = {
 	'thank-you-main-message': 'Lesen Sie unsere Dankesbotschaft',
+	'call-to-action-more-info': 'Weitere Informationen auf unserer Webseite',
 	'call-to-action-button': 'Jetzt Fördermitglied werden',
 	'call-to-action-button-var1': 'Mit 2€ im Monat fördern',
 	'call-to-action-button-var2': 'Mit anderem Betrag fördern',
 	'expand-button-collapse-action': 'Dankestext schließen',
 	'expand-button-expand-action': 'Dankestext lesen',
 	'progress-bar-inner-text': 'Geschafft!',
-	'image-copyright-holder': 'Annika Möller für Wikimedia Deutschland'
+	'image-copyright-holder': 'Annika Möller für Wikimedia Deutschland',
+	'subscribe-title': 'Benötigen Sie weitere Informationen zur Mitgliedschaft?',
+	'subscribe-text': 'Tragen Sie einfach Ihre e-Mail-Adresse ein',
```

Comment:
capitalized noun in German: 
```suggestion
	'subscribe-text': 'Tragen Sie einfach Ihre E-Mail-Adresse ein',
```